### ISSUE TITLE
@note annotation causes error

### ISSUE THREAD

**Issue by mpysiak** [doctrine-extensions/DoctrineExtensions#2879]
#### Package

<details><summary>show</summary>
<p>

```
$ composer show --latest gedmo/doctrine-extensions
name     : gedmo/doctrine-extensions
descrip. : Doctrine behavioral extensions
keywords : Blameable, behaviors, doctrine, extensions, gedmo, loggable, nestedset, odm, orm, sluggable, sortable, timestampable, translatable, tree, uploadable
versions : * 3.17.0
released : 2024-10-06, this week
latest   : 3.17.0 released 2024-10-06, this week
type     : library
license  : MIT License (MIT) (OSI approved) https://spdx.org/licenses/MIT.html#licenseText
homepage : http://gediminasm.org/
source   : [git] https://github.com/doctrine-extensions/DoctrineExtensions.git 38b3737997ce58c1187807d18abc49d0f416b4c4
dist     : [zip] https://api.github.com/repos/doctrine-extensions/DoctrineExtensions/zipball/38b3737997ce58c1187807d18abc49d0f416b4c4 38b3737997ce58c1187807d18abc49d0f416b4c4
path     : /Users/michal/Sylius/Sylius/vendor/gedmo/doctrine-extensions
names    : gedmo/doctrine-extensions

support
email : gediminas.morkevicius@gmail.com
issues : https://github.com/doctrine-extensions/DoctrineExtensions/issues
source : https://github.com/doctrine-extensions/DoctrineExtensions/tree/3.17.0
wiki : https://github.com/Atlantic18/DoctrineExtensions/tree/main/doc

autoload
psr-4
Gedmo\ => src/

requires
behat/transliterator ^1.2
doctrine/collections ^1.2 || ^2.0
doctrine/common ^2.13 || ^3.0
doctrine/deprecations ^1.0
doctrine/event-manager ^1.2 || ^2.0
doctrine/persistence ^2.2 || ^3.0
php ^7.4 || ^8.0
psr/cache ^1 || ^2 || ^3
psr/clock ^1
symfony/cache ^5.4 || ^6.0 || ^7.0

requires (dev)
doctrine/annotations ^1.13 || ^2.0
doctrine/cache ^1.11 || ^2.0
doctrine/dbal ^3.7 || ^4.0
doctrine/doctrine-bundle ^2.3
doctrine/mongodb-odm ^2.3
doctrine/orm ^2.14.0 || ^3.0
friendsofphp/php-cs-fixer ^3.14.0
nesbot/carbon ^2.71 || ^3.0
phpstan/phpstan ^1.11
phpstan/phpstan-doctrine ^1.4
phpstan/phpstan-phpunit ^1.4
phpunit/phpunit ^9.6
rector/rector ^1.1
symfony/console ^5.4 || ^6.0 || ^7.0
symfony/doctrine-bridge ^5.4 || ^6.0 || ^7.0
symfony/phpunit-bridge ^6.0 || ^7.0
symfony/uid ^5.4 || ^6.0 || ^7.0
symfony/yaml ^5.4 || ^6.0 || ^7.0

suggests
doctrine/mongodb-odm to use the extensions with the MongoDB ODM
doctrine/orm to use the extensions with the ORM

conflicts
doctrine/annotations <1.13 || >=3.0
doctrine/dbal <3.7 || >=5.0
doctrine/mongodb-odm <2.3 || >=3.0
doctrine/orm <2.14.0 || 2.16.0 || 2.16.1 || >=4.0
```

</p>
</details>

#### Doctrine packages

<details><summary>show</summary>
<p>

```
$ composer show --latest 'doctrine/*'
Direct dependencies required in composer.json:
doctrine/cache                      2.2.0  2.2.0  PHP Doctrine Cache library is a popular cache implementation that supports many different drivers such as ...
doctrine/collections                1.8.0  2.2.2  PHP Doctrine Collections library that adds additional functionality on top of PHP arrays.
doctrine/common                     3.4.4  3.4.4  PHP Doctrine Common project is a library that provides additional functionality that other Doctrine projec...
doctrine/data-fixtures              1.7.0  1.7.0  Data Fixtures for all Doctrine Object Managers
doctrine/dbal                       3.9.1  4.1.1  Powerful PHP database abstraction layer (DBAL) with many features for database schema introspection and ma...
doctrine/doctrine-bundle            2.13.0 2.13.0 Symfony DoctrineBundle
doctrine/doctrine-migrations-bundle 3.3.1  3.3.1  Symfony DoctrineMigrationsBundle
doctrine/event-manager              2.0.1  2.0.1  The Doctrine Event Manager is a simple PHP event system that was built to be used with the various Doctrin...
doctrine/inflector                  2.0.10 2.0.10 PHP Doctrine Inflector is a small library that can perform string manipulations with regard to upper/lower...
doctrine/migrations                 3.8.1  3.8.1  PHP Doctrine Migrations project offer additional functionality on top of the database abstraction layer (D...
doctrine/orm                        2.19.7 3.2.2  Object-Relational-Mapper for PHP
doctrine/persistence                3.3.3  3.3.3  The Doctrine Persistence project is a set of shared interfaces and functionality that the different Doctri...

Transitive dependencies not required in composer.json:
doctrine/annotations                2.0.2  2.0.2  Docblock Annotations Parser
doctrine/deprecations               1.1.3  1.1.3  A small layer on top of trigger_error(E_USER_DEPRECATED) or PSR-3 logging with options to disable all depr...
doctrine/instantiator               2.0.0  2.0.0  A small, lightweight utility to instantiate objects in PHP without invoking their constructors
doctrine/lexer                      2.1.1  3.0.1  PHP Doctrine Lexer parser library that can be used in Top-Down, Recursive Descent Parsers.
doctrine/sql-formatter              1.5.0  1.5.0  a PHP SQL highlighting library
```

</p>
</details>

#### PHP version

```
$ php -v
PHP 8.3.12 (cli) (built: Sep 24 2024 18:08:04) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.3.12, Copyright (c) Zend Technologies
    with Zend OPcache v8.3.12, Copyright (c), by Zend Technologies
    with Xdebug v3.3.2, Copyright (c) 2002-2024, by Derick Rethans
```

## Subject

`
[Semantical Error] The annotation "@note" in property Gedmo\Loggable\Entity\MappedSuperclass\AbstractLogEntry::$data was never imported. Did you maybe forget to add a "use" statement for this annotation?
`

It is probably caused by [this](https://github.com/doctrine-extensions/DoctrineExtensions/commit/755c0a7c6bfcb5151c14a858a7a96163ee327472#diff-405c4fcd894e288e1071d980a8aa1a8ea0c14c2df421c3ed3bd9f8925ee483d9R88) change

Builds failed in our project: https://github.com/Sylius/Sylius/actions/runs/11207977315/job/31151053094

**Comment by dev-invitario** [comment]
@mpysiak @phansys thanks for fixing this issue so fast.
After running `composer update` is still get the "faulty" 3.17.0 version.

do i need to do something to get the updated version?

**Comment by phansys** [comment]
Hey @dev-invitario!
The issue was fixed by @mpysiak at #2880, but there is no release yet including that fix.
I guess I can release 3.17.1 today.

Thanks for your patience.

**Comment by phansys** [comment]
[3.17.1](https://github.com/doctrine-extensions/DoctrineExtensions/releases/tag/v3.17.1) was released.
